token-minter/main.go

```diff
@@ -139,4 +139,6 @@ func main() {
 	f.WriteString(token.Status.Token)
 
 	fmt.Println("Successfully wrote token to", tokenFile)
+	fmt.Println("Done, starting to sleep")
+	<-ctx.Done()
```

Comment:
why this?

Reply:
Because it is now a sidecar and if it ends the pod will go into crashloop